package.json

```diff
@@ -10,18 +10,16 @@
         "services/*"
     ],
     "devDependencies": {
-        "@dhis2/cli-app-scripts": "^10.2.0",
-        "@dhis2/cli-style": "^10.4.3",
+        "@dhis2/cli-app-scripts": "^12.0.0-alpha.21",
+        "@dhis2/cli-style": "10.4.3",
```

Comment:
this needs to be a fixed version - the latest version that works with TypeScript doesn't work for the setup we have here with nested tsconfig, I will create a separate ticket and investigate it